docs/de/user/installation.rst

```diff
@@ -86,7 +86,7 @@ Führe dieses Kommando aus, um das neueste Paket herunterzuladen und zu entpacke
 
 .. code-block:: bash
 
-    wget http://wllbg.org/latest-v2-package && tar xvf latest-v2-package
+    wget https://wllbg.org/latest-v2-package && tar xvf latest-v2-package
```

Comment:
Thanks, indeed. Could that be fixed by the wllbg.org owner? (the redirection works, but the certificate is invalid for that domain)

I could hardlink the framabag.org destination, but that would defeat the purpose of having the "latest" redirection.